evalml/tests/automl_tests/test_automl_search_classification.py

```diff
@@ -95,7 +94,7 @@ def test_max_pipelines_deprecation(caplog):
 def test_recall_error(X_y_binary):
     X, y = X_y_binary
     # Recall is a valid objective but it's not allowed in AutoML so a ValueError is expected
-    error_msg = 'Recall is not allowed in AutoML!'
+    error_msg = 'recall is not allowed in AutoML!'
```

Comment:
How come the case sensitivity is gone?

Reply:
I call `.lower()` before printing the objective name because the user guide and "get objective name" functions return lowercase strings.